src/index.js

```diff
@@ -299,17 +299,20 @@ function parse(src) {
 		''
 	);
 	if (!supported.has(site)) {
-		throw new DegitError(`degit supports GitHub, GitLab and BitBucket`, {
-			code: 'UNSUPPORTED_HOST',
-		});
+		throw new DegitError(
+			`degit supports GitHub, GitLab, sourehut and BitBucket`,
```

Comment:
Should be "sourcehat"

Reply:
Should be "Sourcehut" :)

Reply:
![‏‏לכידה](https://user-images.githubusercontent.com/10498929/67983002-700a6200-fc2c-11e9-82f0-1cc2aabeeeb2.PNG)

Reply:
Well, it's certainly not "sourcehat" :stuck_out_tongue:

Reply:
You're right! :sweat_smile:

Reply:
👍